int32 label_id
float32 confidence
float64 xmin
float64 ymin
float64 xmax
float64 ymax
geometry_msgs/Vector3 depth